rpc_util.go

```diff
@@ -636,7 +636,7 @@ func recvAndDecompress(p *parser, s *transport.Stream, dc Decompressor, maxRecei
 		return nil, err
 	}
 	if payInfo != nil {
-		payInfo.wireLength = len(d)
+		payInfo.wireLength = len(d) + headerLen
```

Comment:
This field is in sync with `payInfo.uncompressedBytes` (they both don't include the message header).

Add `headerLen` when `HandleRPC` is called?

Reply:
Can do!